### PR TITLE
switch to using curl instead of wget as it is more likely to be installed

### PR DESCRIPTION
**Pull request by jonmort**

### CODE REVIEW

File: manifests/download_and_extract.pp [hunk 1]
```diff
@@ -3,9 +3,9 @@
     $filename = "${dirname}.tar.gz"
     $target = "${installdir}/${filename}"
     exec { "Download ${filename}":
-        command => "wget -q ${baseurl}/${version}/${filename} -O ${target}",
+        command => "curl -s -L ${baseurl}/${version}/${filename} -o ${target}",
         creates => $target,
-        require => File[$installdir]
+        require => [File[$installdir], Package['curl']]
```
Comment (mattohope) [inline manifests/download_and_extract.pp]: there is no package['curl'] defined in this module, should we not add it with a "if ! defined" or "ensure_resource()" condition?

Comment (jonmort) [inline manifests/download_and_extract.pp]: Fair point. I wasn't sure about adding it, and if we do add that should we also add tar?

Comment (jonmort) [inline manifests/download_and_extract.pp]: @mattohope I've added this now.

**Comment by mattohope** [comment]
+1

**Pull request merged**